flowio/tests/flowdata_tests.py

```diff
@@ -1,16 +1,18 @@
+import shutil
 import unittest
 import os
 import io
 import tempfile
 from flowio import FlowData
+import pandas as pd
```

Comment:
FlowIO has a strict requirement to not require external libraries. Looks like this was leftover from your testing.

Reply:
ah, yep, my bad!